### ISSUE TITLE
State that action :resize on a lv is actually a extend

### ISSUE THREAD

**Issue by EugenMayer** [chef-cookbooks/lvm#110]
Took me a while to understand, that you only can extend, since lvextend is used, not lvresize. For one how is trying to actually downsize a lv, this was hard to understand, since

```

lvm_logical_volume 'root' do
  group 'debian-8-vg'
  size '20G'
  action :resize
end
```
will not through any errors, eventhough the lv is 40G big right now, ti will just say "up to date" - all good. But well, of course the LV is not shrinked due to the use of `lvextend` https://github.com/chef-cookbooks/lvm/blob/master/libraries/provider_lvm_logical_volume.rb#L255

There should be a big fat notice or rather rename it to :extend (leave room for a real resize..) and make it for now deprecated to use :resize, while still mapping it to extend until the next major release or whatsoever

**Comment by tas50** [comment]
@EugenMayer Care to open a PR to update the documentation in the readme?

**Comment by irvingpop** [comment]
Closed via #160
